core-feature-pack/common/src/main/resources/content/bin/systemd/generate_systemd_unit.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/sh
+
+# This script is just a helper to generate the systemd unit files for WildFly
+# assuming that the server home will be the one that hold this script file.
+# It also allows to specify the user and group that will run the server.
+
+function print_usage() {
+  echo "INFO: Usage: "
+  echo "${SCRIPT_NAME} -m,--mode standalone|domain [-u,--user user] [-g,--group group] [-c,--confirm y|n]"
+
+  exit 1
+}
+
+SCRIPT_NAME=$(basename "${0}")
+SCRIPT_DIR="$(dirname "$(realpath "${0}")")"
```

Comment:
I think this might be broken:

`"$(dirname "$(realpath "${0}")")"`

I count 2 opening " and 3 closing one. Also, with command substitution double quotes can be omitted (by spec the $() does the same job as ""). So this should be enough, as the most important "" are around the variable replacement, to ensure the result is treated as one arguments (so a whitespace in the file path does NOT break the script):

`$(dirname $(realpath "${0}"))`

Reply:
Done